src/module/chat-message/document.ts

```diff
@@ -171,12 +171,17 @@ class ChatMessagePF2e extends ChatMessage<ActorPF2e> {
         return game.scenes.get(sceneId)?.tokens.get(tokenId) ?? null;
     }
 
+    override getRollData(): Record<string, unknown> {
+        const { actor, item } = this;
+        return { ...actor?.getRollData(), ...(item?.getRollData() ?? { actor }) };
```

Comment:
```suggestion
        return { ...actor?.getRollData(), ...item?.getRollData() };
```

if I recall, actor.getRollData() already has actor in the roll data now as the document itself, as of the v10 release.

Reply:
```js
  /**
   * Prepare a data object which defines the data schema used by dice roll commands against this Actor
   * @returns {object}
   */
  getRollData() {
    return this.system;
  }
```

Reply:
Sorry, ActorPF2e has the override. I should have been more specific.